product-scenarios/3-connecting-web-apis-cloud-services/readme.md

```diff
@@ -0,0 +1,25 @@
+# 3 Connecting Web APIs/Cloud Services
+
+## Business use case narrative
+
+One of the most expected features from an integration solution is the ability to connect to third party applications 
+via public APIs that are exposed by the application developers. WSO2 Integrator brings you in this capability by means 
+of it's inbuilt connectors which, by number exceeds 150. 
+
+A connector is a collection of templates that define operations users can call from their configurations to easily 
+access specific logic for processing messages. Generally, a connector wraps the API of an external service. Each 
+connector is designed to work with a dedicated service hence cannot be used across services. 
```

Comment:
dedicated cloud service

Reply:
modified to synapse configurations

Reply:
not only cloud services, but any third party service